ui/desktop/tests/acceptance/scopes-test.js

```diff
@@ -190,6 +180,7 @@ module('Acceptance | scopes', function (hooks) {
 
   test('can navigate among org scopes via header navigation', async function (assert) {
     assert.expect(3);
+    this.stubClientDaemonSearch('targets', 'targets', 'targets', 'targets');
```

Comment:
From your comment, this will all for 4 calls that will each return targets to the client daemon. Will this search or filter the response according to query params if any are provided?

Reply:
No this won't handle query params, you have to mock the data that should be returned from search/filtering. We're not aiming to test the query params since that's not in our control and we have to assume that the client daemon returns the correct data (though pagination will still be handled because that's not from the search call).

I only did this because search is called 4 times, I thought of maybe providing something to just let it mock all search calls rather than having to be explicit about how many times search will be called but I thought this was fine, let me know if you disagree.